src/NBitcoin/Networks/StratisTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using NBitcoin.Protocol;
+
+namespace NBitcoin.Networks
+{
+    public class StratisTest : StratisMain
+    {
+        public StratisTest()
+        {
+            // The message start string is designed to be unlikely to occur in normal data.
+            // The characters are rarely used upper ASCII, not valid as UTF-8, and produce
+            // a large 4-byte int at any alignment.
+            var messageStart = new byte[4];
```

Comment:
We should maybe make magic bytes simpler? we dont really need the byte array created in advance.